shell_automaton/src/current_head/current_head_effects.rs

```diff
@@ -25,8 +26,18 @@ where
             let chain_id = store.state().config.chain_id.clone();
             let level_override = store.state().config.current_head_level_override;
             let storage_req_id = store.state().storage.requests.next_req_id();
+
+            let latest_context_hashes = match &store.state().protocol_runner {
+                ProtocolRunnerState::Ready(state) => state.latest_context_hashes.clone(),
+                _ => Vec::new(),
```

Comment:
@binier can this situation (we get to this point without the state being `Ready`) happen? if this is not something that should ever happen I would log a warning here just in case (cc @sebastiencs )